### PR TITLE
Add Support for Javascript in cph

### PR DESCRIPTION
**Pull request by lxxonx**
Can you add support for javascript?

### CODE REVIEW

File: src/executions.ts [hunk 1]
```diff
@@ -34,14 +34,15 @@ export const runTestCase = (
         env: {
             DEBUG: 'true',
             CPH: 'true',
+            PATH: process.env.PATH,
         },
     };
 
-    let process: ChildProcessWithoutNullStreams;
+    let testProcess: ChildProcessWithoutNullStreams;
```
Comment (falsetru) [inline src/executions.ts]: Why did you change the variable name?

Comment (lxxonx) [reply src/executions.ts]: I've changed the variable name to import "node:process" module as its default name.

**Comment by agrawal-d** [comment]
Thanks, can you fix the conflicts?

**Comment by agrawal-d** [comment]
Undo unrelated changes in oackage/package-lock files
